packaging/debian/rules

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/make -f
+export DH_VERBOSE = 1
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+export DEB_CFLAGS_MAINT_APPEND  = -Wall
```

Comment:
Remove an extra space maybe? I could see the equal signs aligned with CFLAGS/LDFLAGS, but partial alignment looks rather confusing.

Reply:
removed.